middlewares/users.middleware.js

```diff
@@ -203,22 +215,31 @@ function usersMiddleware(datastore, errorResponse, secret, crypto, CONFIG) {
 
     function createUser(req, res, next) {
         const password = hashPassword(req.body.password);
-
+        var access_date = new Date();
```

Comment:
Is this a mock?

Reply:
It's for setting default access.vendor/customer next payment date, which is used to check access